tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -148,7 +148,7 @@ class WskBasicUsageTests
         }
     }
 
-    it should "show api build using http apihost" in {
+    it should "show api build using http apihost with protocol specified" in {
```

Comment:
yes, this line "I believe the error is thrown blow on line 864 url.Parse." will fail as @dubeejw comments.